ironfish/src/wallet/walletdb/walletdb.ts

```diff
@@ -1297,32 +1297,4 @@ export class WalletDB {
       yield value
     }
   }
-
-  async addParticipantIdentity(
-    account: Account,
-    identity: Buffer,
-    tx?: IDatabaseTransaction,
-  ): Promise<void> {
-    await this.participantIdentities.put([account.prefix, identity], { identity }, tx)
```

Comment:
should we remove this datastore from the walletdb altogether?

maybe in a separate pr